orc8r/gateway/python/magma/ctraced/rpc_servicer.py

```diff
@@ -35,10 +35,13 @@ def add_to_server(self, server):
 
     def StartCallTrace(
         self,
-        _request: StartTraceRequest,
+        request: StartTraceRequest,
         _context,
     ) -> StartTraceResponse:
-        success = self._trace_mgr.start_trace()
+        success = self._trace_mgr.start_trace(
```

Comment:
Let's add some defensive checks here to ensure only valid tshark filter is passed in here.